<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>698</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>DAQ Parser</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>DAQ Parser</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="status_text">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTabWidget" name="streamtabs">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>XFEL_SASE1</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QPushButton" name="filterpb">
             <property name="text">
              <string>Apply Filter</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTableWidget" name="filtertable">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="columnCount">
              <number>0</number>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QTreeWidget" name="treeWidget">
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="indentation">
              <number>25</number>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="animated">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="headerHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Channel Name</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>XFEL_SASE2</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="1">
            <widget class="QTableWidget" name="filtertable_2">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="columnCount">
              <number>0</number>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="filterpb_2">
             <property name="text">
              <string>Apply Filter</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QTreeWidget" name="treeWidget_3">
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="indentation">
              <number>25</number>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="animated">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="headerHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Channel Name</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>XFEL_SASE3</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="1">
            <widget class="QTableWidget" name="filtertable_3">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="columnCount">
              <number>0</number>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QTreeWidget" name="treeWidget_4">
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="indentation">
              <number>25</number>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="animated">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="headerHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Channel Name</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="filterpb_3">
             <property name="text">
              <string>Apply Filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>LINAC</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="2">
            <widget class="QTableWidget" name="filtertable_4">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="columnCount">
              <number>0</number>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="filterpb_4">
             <property name="text">
              <string>Apply Filter</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QTreeWidget" name="treeWidget_5">
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="indentation">
              <number>25</number>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="animated">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="headerHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Channel Name</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>KARABO</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QTableWidget" name="filtertable_5">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="columnCount">
              <number>0</number>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="filterpb_5">
             <property name="text">
              <string>Apply Filter</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QTreeWidget" name="treeWidget_6">
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="indentation">
              <number>25</number>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="animated">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="headerHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="headerStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Channel Name</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Output directory:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="filenameEdit">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="8">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Export files</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="startlabel">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Start time:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="startdate">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="browsepb">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="stoplabel">
            <property name="text">
             <string>Stop time:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QDoubleSpinBox" name="thresholdSpinBox">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string extracomment="GB"/>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>File size threshold (GB):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDateTimeEdit" name="stopdate">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QPushButton" name="hdf5button">
            <property name="text">
             <string>Get Channel List</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="number_of_files">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="timeline">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Stream Catalogue</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0" colspan="3">
        <widget class="QTreeWidget" name="treeWidget_2">
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="loadcataloguepb">
         <property name="text">
          <string>Load stream catalogue</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="filenameEdit2">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Stream:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="browsepb2">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="searchbar">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>browseSlot()</slot>
 </slots>
</ui>
